lib/model/katex.dart

```diff
@@ -523,6 +526,38 @@ class _KatexParser {
                   if (marginLeftEm < 0) throw _KatexHtmlParseError();
                   continue;
                 }
+
+              case 'color':
+                // `package:csslib` parser emits a HexColorTerm for the `color`
+                // attribute. It automatically resolves the named CSS colors to
+                // their hex values. The `HexColorTerm.value` is the hex
+                // encoded in an integer in the same sequence as the input hex
```

Comment:
This is a helpful comment, thanks.

Since the logic in this case gets somewhat long, let's push all that logic, along with this accompanying comment, into a dedicated private helper function. That will help keep it easy to read in isolation, and also keep this surrounding code easy to read when not thinking specifically about colors.